src/js/actions/draganddrop.js

```diff
@@ -89,7 +113,7 @@ define(function (require, exports) {
     /**
     * Check the intersection of the current dragTarget and available drop targets
     *
-    * @param {{x: number, y: number}} point Point from event
+    * @param {Object.<{x: number, y: number}>} point - Point from event
```

Comment:
The previous type annotation was correct: `@param {{x: number, y: number}}`. There are three different notations for objects used in different cases. It's weird, but consistent.